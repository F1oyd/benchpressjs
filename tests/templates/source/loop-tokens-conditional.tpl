<!-- BEGIN rooms.0 -->
  <li>
    <span class="label label-primary">@key</span>
		<!-- IF @value --> &rarr; <span class="label label-default">@value</span><!-- ENDIF @value -->
  </li>
<!-- END rooms.0 -->