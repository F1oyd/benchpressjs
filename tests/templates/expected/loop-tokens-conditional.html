<li>
  <span class="label label-primary">name</span>
   &rarr; <span class="label label-default">Test</span>
</li>
<li>
  <span class="label label-primary">slug</span>
   &rarr; <span class="label label-default">test</span>
</li>
<li>
  <span class="label label-primary">description</span>
   &rarr; <span class="label label-default">testing</span>
</li>
<li>
  <span class="label label-primary">author</span>
   &rarr; <span class="label label-default">1</span>
</li>
<li>
  <span class="label label-primary">private</span>

</li>
<li>
  <span class="label label-primary">clients</span>
</li>
<li>
  <span class="label label-primary">loggedIn</span>
   &rarr; <span class="label label-default">true</span>
</li>